Services/Dashboard/ItemsBlock/classes/class.ilPDSelectedItemsBlockViewSettings.php

```diff
@@ -1,70 +1,59 @@
-<?php
-/******************************************************************************
- *
- * This file is part of ILIAS, a powerful learning management system.
+<?php declare(strict_types=1);
+
+/**
+ * This file is part of ILIAS, a powerful learning management system
+ * published by ILIAS open source e-Learning e.V.
  *
- * ILIAS is licensed with the GPL-3.0, you should have received a copy
- * of said license along with the source code.
+ * ILIAS is licensed with the GPL-3.0,
+ * see https://www.gnu.org/licenses/gpl-3.0.en.html
+ * You should have received a copy of said license along with the
+ * source code, too.
  *
  * If this is not the case or you just want to try ILIAS, you'll find
  * us at:
- *      https://www.ilias.de
- *      https://github.com/ILIAS-eLearning
+ * https://www.ilias.de
+ * https://github.com/ILIAS-eLearning
  *
- *****************************************************************************/
+ *********************************************************************/
 
 /**
  * Class ilPDSelectedItemsBlockViewSettings
  */
 class ilPDSelectedItemsBlockViewSettings implements ilPDSelectedItemsBlockConstants
 {
-    /**
-     * @var int[]
-     */
+    /** @var int[] */
     protected static array $availableViews = [
         self::VIEW_SELECTED_ITEMS,
         self::VIEW_MY_MEMBERSHIPS,
-        self::VIEW_MY_STUDYPROGRAMME
+        self::VIEW_RECOMMENDED_CONTENT,
+        self::VIEW_LEARNING_SEQUENCES,
+        self::VIEW_MY_STUDYPROGRAMME,
     ];
 
-    /**
-     * @var string[]
-     */
-    protected static array $availableSortOptions = [
-        self::SORT_BY_LOCATION,
-        self::SORT_BY_TYPE,
-        self::SORT_BY_START_DATE
+    /** @var string[] */
+    protected static array $viewNames = [
+        self::VIEW_SELECTED_ITEMS => 'favorites',
+        self::VIEW_MY_MEMBERSHIPS => 'memberships',
+        self::VIEW_MY_STUDYPROGRAMME => 'study_programme',
+        self::VIEW_RECOMMENDED_CONTENT => 'recommended_content',
+        self::VIEW_LEARNING_SEQUENCES => 'learning_sequences'
     ];
 
-    /**
-     * @var string[]
-     */
+    /** @var string[] */
     protected static array $availablePresentations = [
         self::PRESENTATION_LIST,
         self::PRESENTATION_TILE
     ];
 
-    /**
-     * @var array[]
-     */
-    protected static array $availableSortOptionsByView = [
-        self::VIEW_SELECTED_ITEMS => [
-            self::SORT_BY_LOCATION,
-            self::SORT_BY_TYPE,
-            self::SORT_BY_ALPHABET,
-        ],
-        self::VIEW_MY_MEMBERSHIPS => [
+    /** @var string[] */
+    protected static array $availableSortOptions = [
             self::SORT_BY_LOCATION,
             self::SORT_BY_TYPE,
             self::SORT_BY_START_DATE,
             self::SORT_BY_ALPHABET,
-        ],
-        self::VIEW_MY_STUDYPROGRAMME => []
     ];
 
-    /**
-     * @var array[]
-     */
+    /** @var array[] */
```

Comment:
Please make the type annotation of `$availablePresentationsByView` more specific: `array[]` -> `array<int, string[]>`